middleware/requiresAuth.js

```diff
@@ -133,7 +126,7 @@ module.exports.custom = function custom(func) {
 
     const { idTokenClaims } = req.oidc;
 
-    return func(req, idTokenClaims);
+    return !func(req, idTokenClaims);
```

Comment:
I thought `claimCheck(() => true)` should mean allow, `claimCheck(() => false)` should mean 401